contracts/extensions/ClaimExtension.sol

```diff
@@ -0,0 +1,690 @@
+    function batchAbsorb(
+        ISetToken _setToken,
+        address[] memory _tokens
+    )
+        external
+        onlyValidAbsorbCaller(_setToken)
+        onlyAllowedAssets(_setToken, _tokens)
+    {
+        _batchAbsorb(_setToken, _tokens);
+    }
+
+    /**
+     * ONLY VALID ABSORB CALLER: Absorb specified token into position. If airdropFee defined, send portion to feeRecipient and portion to
+     * protocol feeRecipient address. Callable only by operator unless anyoneAbsorb is true on the AirdropModule.
+     *
+     * @param _setToken                 Address of SetToken
+     * @param _token                    Address of token to absorb
+     */
+    function absorb(
+        ISetToken _setToken,
+        IERC20 _token
+    )
+        external
+        onlyValidAbsorbCaller(_setToken)
+        onlyAllowedAsset(_setToken, address(_token))
+    {
+        _absorb(_setToken, _token);
+    }
+
+    /**
+     * ONLY OWNER: Adds new tokens to be added to positions when absorb is called.
+     *
+     * @param _setToken                 Address of SetToken
+     * @param _airdrop                  Component to add to airdrop list
+     */
+    function addAirdrop(
+        ISetToken _setToken,
+        IERC20 _airdrop
+    )
+        external
+        onlyOwner(_setToken)
+    {
+        bytes memory callData = abi.encodeWithSelector(
+            IAirdropModule.addAirdrop.selector,
+            _setToken,
+            _airdrop
+        );
+        _invokeManager(_manager(_setToken), address(airdropModule), callData);
+    }
+
+    /**
+     * ONLY OWNER: Removes tokens from list to be absorbed.
+     *
+     * @param _setToken                 Address of SetToken
+     * @param _airdrop                  Component to remove from airdrop list
+     */
+    function removeAirdrop(
+        ISetToken _setToken,
+        IERC20 _airdrop
+    )
+        external
+        onlyOwner(_setToken)
+    {
+        bytes memory callData = abi.encodeWithSelector(
+            IAirdropModule.removeAirdrop.selector,
+            _setToken,
+            _airdrop
+        );
+        _invokeManager(_manager(_setToken), address(airdropModule), callData);
+    }
+
+    /**
+     * ONLY OWNER: Update whether manager allows other addresses to call absorb.
+     *
+     * @param _setToken                 Address of SetToken
+     */
+    function updateAnyoneAbsorb(
+        ISetToken _setToken,
+        bool _anyoneAbsorb
+    )
+        external
+        onlyOwner(_setToken)
+    {
+        bytes memory callData = abi.encodeWithSelector(
+            IAirdropModule.updateAnyoneAbsorb.selector,
+            _setToken,
+            _anyoneAbsorb
+        );
+        _invokeManager(_manager(_setToken), address(airdropModule), callData);
+    }
+
+    /**
+     * ONLY OWNER: Update address AirdropModule manager fees are sent to.
+     *
+     * @param _setToken             Address of SetToken
+     * @param _newFeeRecipient      Address of new fee recipient
```

Comment:
We may want an endpoint to `distributeFees` to the DelegatedManager, similar to the [IssuanceExtension](https://github.com/SetProtocol/set-v2-strategies/blob/24d80780e18b207be70e6e5a6f75609a83375d42/contracts/extensions/IssuanceExtension.sol#L87), but with the reward token as input.

Reply:
My thinking here was that we accrue to the `DelegatedManager` first, and then distribute to the `owner` and `methodologist` according to the fee split. 

With the IssuanceExtension we
- accrue mint/redeem fees to the DelegatedManager everytime someone calls mint/redeem
- distribute via `IssuanceExtension.distributeFees()`

We would follow a similar pattern with the ClaimExtension with
- accrue absorb fees to the DelegatedManager everytime someone calls absorb
- distribute via `ClaimExtension.distributeFees(rewardToken)`

Reply:
Ok great, yes agree it makes sense that this would have the same interface & behavior as IssuanceExtension.